seed/challenges/01-responsive-web-design/basic-html-and-html5.json

```diff
@@ -759,18 +759,15 @@
       "id": "bad88fee1348bd9aedf08816",
       "title": "Link to Internal Sections of a Page with Anchor Elements",
       "description": [
-        "In addition to creating external links, anchor elements can also be used to create internal links, which are links that jump to different sections within a webpage.",
-        "The format is similar to an external link except instead of a URL, you'll use the <code>#</code> and a word to describe the section you want to jump to.",
-        "Here's an example:",
-        "<code>&lt;a href=\"#contact\"&gt;Go to contact section&lt;/a&gt;</code>",
-        "Next, you'll want to create a corresponding anchor link in the HTML where you want the internal link to take users. Instead of using the <code>href</code> attribute, you'll use the <code>name</code> attribute. Here's an example:",
-        "<blockquote>&lt;h1&gt;Contact&lt;/h1&gt;<br>&lt;a name=\"contact\"&gt;&lt;/a&gt;</blockquote>",
-        "Now when users click the \"Go to contact section\" link, they'll be taken to the section of the webpage with the anchor that has the <code>name</code> attribute \"contact\".",
+        "Anchor or <code>a</code> elements can also be used to create an internal link to jump to different section within a webpage.",
+        "To create an internal link, the <code>a</code> tag's <code>href</code> attribute contains the hash symbol <code>#</code> and the <code>id</code> of the target element to jump to.  An <code>id</code> is an attribute that uniquely describes an element.",
```

Comment:
Thanks! I knew that sentence was awkward, your version sounds much better.   What do you think about my second sentence about id.     I felt the need to include because id's hadn't been brought up in prior lessons yet.

> An id is an attribute that uniquely describes an element.